<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userSetDialog</class>
 <widget class="QDialog" name="userSetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>142</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nicknameLabel">
       <property name="text">
        <string>昵　　称：</string>
       </property>
       <property name="buddy">
        <cstring>nicknameEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="pwAgainLabel">
       <property name="text">
        <string>确认密码：</string>
       </property>
       <property name="buddy">
        <cstring>pwAgainEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="passwordEdit">
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="pwAgainEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="passwordLabel">
       <property name="text">
        <string>新 密 码：</string>
       </property>
       <property name="buddy">
        <cstring>passwordEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nicknameEdit"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="nicknameSetButton">
       <property name="text">
        <string>更新昵称</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="passwordSetButton">
       <property name="text">
        <string>更改密码</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="exitButton">
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nicknameEdit</tabstop>
  <tabstop>nicknameSetButton</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>pwAgainEdit</tabstop>
  <tabstop>passwordSetButton</tabstop>
  <tabstop>exitButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>userSetDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
